cypher-shell/src/main/java/org/neo4j/shell/CommandReader.java

```diff
@@ -0,0 +1,41 @@
+package org.neo4j.shell;
+
+import jline.console.ConsoleReader;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommandReader {
+    private final ConsoleReader reader;
+    private final Shell shell;
+    static final Pattern MULTILINE_BREAK = Pattern.compile("\\\\\\s*$");
```

Comment:
can you please give a plain simple example what you try to match with this expression? not sure it's currently to what it suppose to do